content/en/examples/controllers/daemonset.yaml

```diff
@@ -35,6 +35,9 @@ spec:
         volumeMounts:
         - name: varlog
           mountPath: /var/log
+      # it may be desirable to set a high priority class to ensure that a DaemonSet Pod
+      # preempts running Pods
+      # priorityClassName: system-cluster-critical
```

Comment:
How about:
```suggestion
      # priorityClassName: important
```

Reply:
This suggestion is reasonable. @tzneal Please take a look at it.